vivisect/qt/memory.py

```diff
@@ -42,6 +42,7 @@ def __init__(self, *args, **kwargs):
         self.addHotKey('n', 'viv:setname')
         self.addHotKey('g', 'viv:getlocation')
         self.addHotKey(';', 'viv:comment')
+        self.addHotKey(':', 'viv:commentmulti')
```

Comment:
if copy/pasting, we already store multi-line comments.  this one just lets us control which way we interact with the comment.  
these are separate for a UX reasons:  
* comment editing is finished with <Return> key
* multi-line comment editing is less friendly (and probably worse because i'm not a gui developer).  i haven't made it this way, but ideally multi-line comment editing would finish with <ctrl-Return> (i'm not sure of the best way to elegantly do that)

i'm not married to the way i have it here.  i'm not sure we could easily "autodetect" it since that sounds like a chicken/egg problem (i know how to paste in multi-line comments, but if i'm starting from scratch typing a new comment, how do i detect multi-line without losing the <Return> behavior.  
if i were a better GUI programmer, i'd add a checkbox that switches it between one-/multi-line, but i'm not sure how to switch between the two different core GUI widgets.

it'd be really nice to have some help from a knowledgeable PyQT dev :)